org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/parser/antlr/AntlrToolFacade.java

```diff
@@ -122,13 +127,34 @@ public boolean download() {
 			}
 			log.info("Finished downloading.");
 		} catch (IOException e) {
-			System.err.println("Downloading ANTLR parser generator failed: " + e.getMessage());
-			System.err.println("Please install the feature 'Xtext Antlr SDK' manually using the external updatesite:");
-			System.err.println();
-			System.err.println("\t\t'http://download.itemis.com/updates/'."); 
-			System.err.println();
-			System.err.println("(see http://www.eclipse.org/Xtext/download.html for details)");
-			return false;
+			try {
+				log.warn("Downloading file from '"+secureDownloadURL+"' failed.", e);
+				log.info("Downloading file from '"+downloadURL+"' ...");
+				BufferedInputStream in = new BufferedInputStream(new URL(downloadURL).openStream());
+				try {
```

Comment:
copy & paste is ugly but i dont see a nice option for reuse here

Reply:
I think the fallback to http if https failed for some reason is not something that should be done transparently.
Can we assume that https will usually work? Should clients that cannot use https change the download location in their mwe2 file instead?

Reply:
there is no api to store anything in mwe yet

Reply:
i also want to keep the fallback in case people dont test early

Reply:
Ok